packages/datadog-plugin-bluebird/test/index.spec.js

```diff
@@ -3,3 +3,5 @@
 const assertPromise = require('../../dd-trace/test/plugins/promise')
 
 assertPromise('bluebird')
+
+assertPromise('bluebird', bluebird => { return bluebird.getNewLibraryCopy() }, '^2.11.0 || ^3.4.1')
```

Comment:
Small nit, but the `return` can be avoided by removing the curly braces.

Reply:
updated